docs/SUMMARY.md

```diff
@@ -22,6 +22,7 @@
 * [quilt3.Bucket](api-reference/Bucket.md)
 * [CLI, environment](api-reference/cli.md)
 * [Known limitations](api-reference/limitations.md)
+* [Self-signed certificates](api-reference/self-signed-certificates.md)
```

Comment:
```suggestion
* [Custom SSL certs](api-reference/self-signed-certificates.md)
```

Reply:
Updated to:

`* [Custom SSL certificates](api-reference/custom-ssl-certificates.md)`

Better to use the full word instead of an abbreviation for SEO.